xcp_d/tests/test_confounds.py

```diff
@@ -52,7 +52,7 @@ def test_custom_confounds(fmriprep_with_freesurfer_data, tmp_path_factory):
         custom_confounds=custom_confounds_file,
     )
     # We expect n params + 2 (one for each condition in custom confounds)
-    assert combined_confounds.shape == (N_VOLUMES, 26)
+    assert combined_confounds.shape == (N_VOLUMES, 28)
```

Comment:
Why did this change?

Reply:
I moved the addition of the linear trend and intercept columns into `load_confound_matrix`. Before they were added on afterwards, so the output of `load_confound_matrix` has two more columns.